scripts/CompareFitMinimizers/results_output.py

```diff
@@ -120,6 +126,22 @@ def print_group_results_tables(minimizers, results_per_test, problems_obj, group
         print(tbl_runtime_summary)
 
 
+def save_table_to_file(table_data, errors, group_name, metric_type, file_extension):
```

Comment:
Looks good with refactoring this code into a new method. Could you add documentation please to this method please including for its arguments